src/common/utils/validator.ts

```diff
@@ -93,3 +93,16 @@ export const isEnglish = (str: string): boolean => {
   }
   return /^[a-zA-Z0-9]*$/.test(str)
 }
+
+/**
+ * Validate circle name. It only accepts alphabets, chinese characters and _.
+ */
+export const isValidCircleName = (name: string, size = 20): boolean => {
+  // TODO: add common disallowed names
+  if (!name || name.length < 2 || name.length > size) {
+    return false
+  }
+  return /^[_A-Za-z\u4E00-\u9FFF\u3400-\u4DFF\uF900-\uFAFF\u2e80-\u33ffh]*$/.test(
```

Comment:
`/^[a-zA-Z0-9]*$/`?

Reply:
Number is not allowed written on Figma. 😂

Reply:
@devformatters I saw displayName is reused by user and circle, and the validation rule of circle name is also similar to username?